packages/user-mgnt/src/features/getAvatar/handler.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as Hapi from '@hapi/hapi'
+import * as Joi from 'joi'
+import { unauthorized } from '@hapi/boom'
+import User, { IUserModel } from '@user-mgnt/model/user'
+import { AVATAR_API } from '@user-mgnt/constants'
+import fetch from 'node-fetch'
+
+interface IAvatarPayload {
+  userId: string
+}
+
+export default async function getUserAvatar(
```

Comment:
Can you make sure this endpoint returns the right Content-Type for the image? This is what I'm currently getting for a PNG image

```
content-type: application/octet-stream
vary: origin
access-control-expose-headers: WWW-Authenticate,Server-Authorization
cache-control: no-cache
content-length: 1193
Date: Mon, 23 May 2022 07:11:07 GMT
Connection: keep-alive
Keep-Alive: timeout=5
```

Reply:
Fixed it. Thanks for pointing this out.